test/JDBC/input/pg_hint_plan/BABEL-3513-vu-prepare.mix

```diff
@@ -46,9 +46,5 @@ go
 set babelfish_showplan_all off;
 go
 
--- psql
-do $$
-begin
-    EXECUTE format('ALTER DATABASE %s SET babelfishpg_tsql.enable_pg_hint = on', CURRENT_DATABASE());
-end$$;
+exec sp_babelfish_configure 'enable_pg_hint', 'on', 'server'
```

Comment:
I forgot to mention https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/test/JDBC/expected/BABEL-3513-vu-verify.out#L94-L99